### ISSUE TITLE
can't add a new vdl with a new copyright year

### ISSUE THREAD

**Issue by jhahn21** [vanadium/issues#1176]
When adding a new vdl file with a new copyright year to an existing package, it fails:

```
#0 v.io/x/ref/lib/discovery/errors.vdl:1:1 all files in a package must have the same file doc (the comment on the first line of each *.vdl file that isn't package doc)
   (run with "vdl -v" for verbose logging or "vdl help" for help)
```

Shouldn't we loosen the compileFileDoc() to ignore copyright year?

**Comment by afandria** [comment]
One of the reasons this requirement is in place is because JS places all *.vdl files from the same vdl package into the same file.

Because of that, we could do these things:
- Update all vdl files in that package to have the same copyright year (2015 or 2016)
- Modify the vdl generator code to parse the copyright year out of the header and use the highest value for JS.

**Comment by tatatodd** [comment]
I agree with Alex's analysis.  One minor fix: I think we might want to use the _lowest_ value for JS.  E.g. if we have vdl files written from 2015 and 2016, the generated file copyright should say 2015, since artifacts from that file were first written then.

That said, for now I actually think we should just update the vdl files to all say 2015 (or whatever the earliest year was).  That seems simplest.

**Comment by caprita** [comment]
Do we even need a copyright notice at all, especially for generated files? http://www.copyright.gov/circs/circ03.pdf seems to suggest copyright notices are no longer needed in order to protect creative work, and google3 removed the requirement to add such notices cca 2012.
